.env

```
@@ -0,0 +1,12 @@
+DOCKER_UID=1000
```

```suggestion
DOCKER_UID=1000
```
usually we don't keep `.env` files in the repo. Is there any specific reason for this change?
The idea behind using `.env.sample` was to force user to look at the file and make necessary changes.

---

I wanted to avoid the repeated `${MINE_NAME:-biotestmine}` but I can put the defaults back in the compose file no problem. I don't see` .env.sample`.

---

No worries! I guess it's fine here. We can make changes later if needed.